.github/workflows/internal-airbyte-ci-tests.yml

```
@@ -1,4 +1,4 @@
-name: Internal Poetry packages CI
+name: Internal CI Package Checks
```

```suggestion
name: Internal CI Tooling Checks
```
Maybe this is slightly more clear...